### ISSUE TITLE
A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_ScreenReader: Tool tip assigned to the texts is not announced by screen reader.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9259]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #Win32; #MAS1.3.1; #MAS1.1.1;  #Benchmark; #A11y_SQLAzureDataStudio; #A11ySev2; #NVDA; #JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen Reader: NVDA(2019.3.1),JAWS
MAS reference: MAS1.3.1, MAS1.1.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install.
5. Launch screen reader and navigate to the content present on the top of the screen.
6. Check if tool tip is visible after focusing the control/text and tool tip is read by the screen reader.

**Actual:
Tool tip is not read by both NVDA and JAWS screen readers.

**Expected:**
Tool tip assigned to all the contents/controls should be read by screen reader.

**User Impact:**
Keyboard as well as screen reader users will not receive the tool tip information that is assigned to the texts/controls present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:
MAS 1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**MAS 1.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachment link reference:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-atkulk_microsoft_com/EbWzICWOKqJOmerxvRr_jn8BphyPgZn63etDHKkmGYUlLg?e=suGVd1

**Comment by alanrenmsft** [comment]
vscode issue, umbrella bug: https://github.com/microsoft/vscode/issues/52016

**Comment by Harika1702** [comment]
<img width="960" alt="9259_A11y_Issue fixed" src="https://user-images.githubusercontent.com/100270074/217818173-29e69ef3-369a-47fa-9087-b51a6aae1d42.png">
Verified this issue in the below environment.
App name: Azure data studio.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
